lib/kubeclient/common.rb

```diff
@@ -259,10 +259,11 @@ def watch_entities(resource_name, options = {})
       Kubeclient::Common::WatchStream.new(uri, http_options(uri))
     end
 
-    # Accepts the following string options:
+    # Accepts the following options:
     #   :namespace - the namespace of the entity.
     #   :label_selector - a selector to restrict the list of returned objects by their labels.
     #   :field_selector - a selector to restrict the list of returned objects by their fields.
```

Comment:
worth clarifying the selectors are strings. examples from tests:
label_selector: "name=redis-master"
field_selector: "involvedObject.name=redis-master"